apollo-runtime/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
   compile dep.supportAnnotations
   compile dep.okHttp
   compile dep.moshi
-  provided project(":apollo-api")
+  compile project(":apollo-api")
```

Comment:
iirc, at that point, the `apollo-api` dependency wasn't really needed at all and we weren't' decided on how runtime would look like yet, we only had an `Apollo` class that wraps `Retrofit` and a `Utils` class. At that time, we also changed the plugin to add `apollo-api` deps to the compile dependency set.

We're gonna have to change it to `compile` anyways (#311). I guess for now we can keep it this way and I'll revisit this when I re-open #311.